server/src/main/java/io/littlehorse/common/ServerContextImpl.java

```diff
@@ -0,0 +1,30 @@
+package io.littlehorse.common;
+
+import io.littlehorse.common.model.getable.global.acl.PrincipalModel;
+
+public final class ServerContextImpl implements ServerContext {
+
+    private final String tenantId;
+
+    private final ServerContext.Scope scope;
+
+    public ServerContextImpl(final String tenantId, final ServerContext.Scope scope) {
+        this.tenantId = tenantId;
+        this.scope = scope;
+    }
+
+    @Override
+    public Scope scope() {
+        return scope;
+    }
+
+    @Override
+    public PrincipalModel principal() {
```

Comment:
Two questions:
1. Are we going to standardize on using `getFoo()` or `foo()`, or does it not matter? I personally don't care, I'm not a java guy.
2. Why `null`?

Reply:
`foo()` is preferred in this case